language/analyze/README.md

```diff
@@ -17,15 +17,15 @@
 ## Run the sample
 
 ```bash
-go run analyze.go <command> <text>
+go build -o analyze && ./analyze <command> <text>
```

Comment:
Can we just omit the -o flag? The binary name will be analyze anyways.

Reply:
yeah, i thought to make it a bit more explicit. done